man/systemd-nspawn.xml

```diff
@@ -477,6 +466,23 @@
       </varlistentry>
 
       <varlistentry>
+        <term><option>-U</option></term>
+
+        <listitem><para>If the kernel supports the user namespaces feature, equivalent to
+        <option>--private-users=pick --private-users-chown</option>, otherwise equivalent to
+        <option>--private-users=no</option>.</para>
+
+        <para>Note that <option>-U</option> is the default if the <filename>systemd-nspawn@.service</filename> template unit
+        file is used.</para>
+
+        <para>Note: it is possible to undo the effect of <option>--private-users-chown</option> (or <option>-U</option>)
+        by redoing the operation with the first UID of 0:</para>
```

Comment:
Considering how simple this is, I don't see a good reason not to do this. I only wonder what the best way to expose this would be. IMHO it should be a separate command-like option that would take the same argument as `--private-users`. I'm only not sure whether this should be in systemd-nspawn or in one of the other tools.